packages/clay-form/src/MultiSelectInput.tsx

```diff
@@ -115,17 +136,21 @@ const ClayMultiSelect: React.FunctionComponent<IProps> = ({
 	useLayoutEffect(() => {
 		if (sourceItems) {
```

Comment:
We probably don't even need this conditional anymore.

```suggestion
		if (sourceItems.length > 0) {
```